cl.conf

```diff
@@ -36,6 +36,20 @@ systemd:
         RemainAfterExit=true
         User=core
 
+        [Install]
+        WantedBy=multi-user.target
```

Comment:
Why did you add this?

Reply:
It's already present in `BRICK/develop` branch, not sure why it shows as new addition

```
    - name: install-minikube.service
      enable: true
      contents: |
        [Unit]
        Description=Install minikube
        ConditionPathExists=!/usr/local/bin/localkube

        [Service]
        Type=oneshot
        ExecStart=/opt/minikube/install-minikube.sh
        RemainAfterExit=true
        User=core

        [Install]
        WantedBy=multi-user.target
```

* git log shows it correctly
```
| diff --git a/cl.conf b/cl.conf
| index ad99030..df9e3d3 100644
| --- a/cl.conf
| +++ b/cl.conf
| @@ -38,6 +38,20 @@ systemd:
|
|          [Install]
|          WantedBy=multi-user.target
| +    - name: install-bricks.service
| +      enable: true
| +      contents: |
| +        [Unit]
| +        Description=Install bricks
| +
| +        [Service]
| +        Type=oneshot
| +        ExecStart=/opt/bricks/install-bricks.sh
| +        RemainAfterExit=true
| +        User=core
| +
| +        [Install]
| +        WantedBy=multi-user.target
```